tests/library_checker_aizu_tests/handmade_tests/dsu_size.test.cpp

```diff
@@ -1,6 +1,7 @@
-#define PROBLEM "https://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=ITP1_1_A"
+#define PROBLEM \
+  "https://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=ITP1_1_A"
 #include "../template.hpp"
-
+// no re-order
```

Comment:
this is annoying, but it keeps moving this when I save it with `clang-format` auto-formatting